libs/hpc-ui/src/lib/components/main-navigation.tsx

```diff
@@ -60,6 +59,12 @@ const Nav = styled.nav`
       align-items: center;
       color: #221e1f;
 
+     .${CLS.LOGO_CONTAINER} 
+        position: relative;
+        z-index: 1;
+        display: flex;
+        height: 36px;
+
```

Comment:
This is very likely invalid CSS just ignored by the parsers. You're missing curly braces. But even if this is a valid CSS, it is completely uncalled for, because it is a copy paste of a selector that you have below.